src/encoded/audit/experiment.py

```diff
@@ -1730,6 +1731,7 @@ def audit_experiment_biosample_characterization(value, system, excluded_types):
         ):
             continue
         biosample = rep['library']['biosample']
+        ontology = biosample['biosample_ontology']['@id']
```

Comment:
this is actually a little dangerous unless you want a 500 error if the keys don't exist.  I think they are strictly require here so it's fine.

Reply:
could wrap that in a try... except KeyError.